lib/rules/indent.js

```diff
@@ -1217,8 +1217,10 @@ module.exports = {
                 const firstNonObjectToken = sourceCode.getFirstTokenBetween(node.object, node.property, astUtils.isNotClosingParenToken);
                 const secondNonObjectToken = sourceCode.getTokenAfter(firstNonObjectToken);
 
-                const tokenBeforeObject = sourceCode.getTokenBefore(node.object, token => astUtils.isNotOpeningParenToken(token) || parameterParens.has(token));
-                const firstObjectToken = tokenBeforeObject ? sourceCode.getTokenAfter(tokenBeforeObject) : sourceCode.ast.tokens[0];
+                const objectParenCount = sourceCode.getTokensBetween(node.object, node.property).findIndex(astUtils.isNotClosingParenToken);
```

Comment:
Wondering if something like `sourceCode.getTokensAfter(node.object, astUtils.isClosingParenToken).length` might be more readable here? As it is, I could see some developers being confused about how the index returned from `findIndex` would correspond to a count of closing parens.